api_utils/ask_bot.py

```
@@ -0,0 +1,61 @@
+import sys
+from . import printe
+
+yes_answer = ["y", "a", "", "Y", "A", "all", "aaa"]
+
+Save_or_Ask = {}
+
+class ASK_BOT:
+    def __init__(self):
+        pass
+
+    def ask_put(self, nodiff=False, newtext="", text="", message="", job="Genral", username="", summary=""):
```

Pylint (3.3.7)

[convention] Line too long (108/100)

(C0301)